### DISCUSSION TITLE
Bonsai-Flycapture

### DISCUSSION THREAD

**Discussion by fribrephotometrysetup** [bonsai-rx/bonsai#1735] [Q&A]
Hi,

We are trying to set up fibre photometry (Neurophotometrics) and are having issues with Bonsai. We have installed Flycapture version 2.11.3.60 and Bonsai version 2.4. When we try to open our bonsai file in the bonsai software, we see an error message, (This is a proxy for the unknown type, install any missing packages - refer to image). Therefore, we referred online and tried to add the 'Bonsai-starter pack' which showed operation failed - 'Part URI cannot start with two forward slashes' (refer to image). We are stuck here and do not know how to navigate further. Have anyone faced similar issues before? Any help will be greatly appreciated. Thank you!!
![IMG_7100](https://github.com/bonsai-rx/bonsai/assets/166771583/4ae69419-5128-4040-a500-47cb1d32a66b)
![IMG_7101](https://github.com/bonsai-rx/bonsai/assets/166771583/d39f3073-0261-497a-b0fd-8f58ac48ffaa)

**Comment by bruno-f-cruz** [comment]
Heya. 
Can you attach your Bonsai.Config and Nuget.Config files that can be found in the Bonsai installation directory? (https://bonsai-rx.org/docs/articles/environments.html)
Additionally, can you attach the code you are trying to run? thanks!

**Reply by fribrephotometrysetup** [reply]
Hi,

Thanks for the response, This folder has NuGet config file, Bonsai config file, and the code we are trying to run.
[Bonsai-config files.zip](https://github.com/bonsai-rx/bonsai/files/14974509/Bonsai-config.files.zip)

**Reply by bruno-f-cruz** [reply]
The following environment seems to be working on my computer (make sure to use bonsai 2.8.2)
[bonsai.zip](https://github.com/bonsai-rx/bonsai/files/14974612/bonsai.zip)

**Reply by fribrephotometrysetup** [reply]
Hi,

Thanks again for sharing the files, we installed the newer version of Bonsai and used your files, now it seems to be working alright!!
